integration_tests/dbt_project.yml

```
@@ -26,6 +26,11 @@ vars:
     usage_source_type_device_report_identifier: "usage_source_type_device"
     usage_territory_source_type_report_identifier: "usage_territory_source_type"
 
+models:
+  +persist_docs:
+    relation: true
+    columns: true
```

Can you also add the databricks conditional logic to be consistent across implementations.
```suggestion
models:
  +persist_docs:
    relation: "{{ false if target.type in ('spark','databricks') else true }}"
    columns: "{{ false if target.type in ('spark','databricks') else true }}"
```

---

Added!